src/cps/env.cpp

```diff
@@ -14,6 +14,12 @@ namespace cps {
         if (const char * env_c = std::getenv("CPS_PATH")) {
             env.cps_path = std::string(env_c);
         }
+        if (const char * env_c = std::getenv("CPS_PREFIX_PATH")) {
```

Comment:
Oops. I should have said this before, but `env_c` could equal `nullptr` if `CPS_PREFIX_PATH` is not defined. Same for the other environment variables.

That also means we're probably missing test cases for running `cps-config` with those environment variables undefined. Maybe we can provide (platform specific?) default values for these variables. But we definitely don't want to construct strings from nullptr.